support/ci/deploy_mac.sh

```diff
@@ -95,14 +95,15 @@ echo "Built new version of hab"
 ln -sf /usr/local/bin/gtar /usr/local/bin/tar
 hash -r
 
-cd ${bootstrap_dir}
+cd "${bootstrap_dir}"
 
 echo "Publishing hab to $BINTRAY_REPO"
-release=$(find $mac_dir/results -name core-hab-0*.hart | sort -n | tail -n 1)
+# shellcheck disable=2061
```

Comment:
```
In ./support/ci/deploy_mac.sh line 101:
release=$(find "$mac_dir"/results -name core-hab-0*.hart | sort -n | tail -n 1)
                                        ^-- SC2061: Quote the parameter to -name so the shell won't interpret it.
```
This one seems pretty clear that we want the glob interpreted by `find` rather than the shell, so it should be:
```bash
find "$mac_dir"/results -name "core-hab-0*.hart"
```

Reply:
Yes, correct.